component/authenticator/src/test/java/org/wso2/carbon/extension/identity/authenticator/duo/test/DuoAuthenticatorTest.java

```diff
@@ -272,6 +280,7 @@ public void testGetConfigurationProperties() {
 
     @ObjectFactory
     public IObjectFactory getObjectFactory() {
+
         return new PowerMockObjectFactory();
     }
 }
```

Comment:
add a blank line

Reply:
Fixed in https://github.com/wso2-extensions/identity-outbound-auth-duo/pull/22/commits/1f21dce91d956f7f59b80e69032e5d854eb604c3